examples/react/css/src/breadcrumbs.css

```diff
@@ -0,0 +1,38 @@
+.ais-Breadcrumb-list {
+  @media screen and (min-width: 768px) {
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    row-gap: var(--ais-spacing-third);
+  }
+}
+
+.ais-Breadcrumb-item {
+  margin-block-end: var(--ais-spacing-half);
+
+  @media screen and (min-width: 768px) {
+    margin-block-end: 0;
+  }
+}
+
+.ais-Breadcrumb-item--selected {
+  color: rgb(var(--ais-text-color-rgb));
+  font-weight: var(--ais-font-weight-semibold);
+}
+
+.ais-Breadcrumb-separator {
+  color: rgb(var(--ais-muted-color-rgb));
+  font-weight: normal;
+  margin: 0 var(--ais-spacing-third);
+}
+
+.ais-Breadcrumb-link {
+  color: rgb(var(--ais-muted-color-rgb));
+  transition: color 0.2s ease-out;
```

Comment:
You can use the CSS variables `--ais-transition-duration` and `--ais-transition-timing-function`